airflow/contrib/hooks/aws_hook.py

```diff
@@ -14,11 +14,64 @@
 
 
 import boto3
+import configparser
 
 from airflow.exceptions import AirflowException
 from airflow.hooks.base_hook import BaseHook
 
 
+def _parse_s3_config(config_file_name, config_format='boto', profile=None):
```

Comment:
Where is the test to this function?

Reply:
There is none, it's legacy code I moved from the S3hook